### ISSUE TITLE
Questions about experimental results

### ISSUE THREAD

**Issue by zw615** [divyam02/dafrcnn-pytorch#9]
Hi, I have some questions regarding the performance of your code.
The original paper's performance ( using the  Cityscapes training set  as the source domain and the  Foggy Cityscapes validation set as the target domain) is  lower than your code.

original:
![图片](https://user-images.githubusercontent.com/26880977/58847384-5680e080-86b5-11e9-9be5-625763fb300c.png)

yours:
![图片](https://user-images.githubusercontent.com/26880977/58847839-f0955880-86b6-11e9-9cde-1c0d53d8c1a2.png)

I wonder why. Did you add some additional tricks or the implementation of original paper is buggy? Also, can your result be reproduced stably?
Wonderful work, by th way. Thanks a lot

**Comment by divyam02** [comment]
Hi! I have been tinkering around with this for some time and this was
probably due to a different code used for the domain classifier. You would
notice some categories perform much worse than the bench mark while some
outperform them. I will release a more streamlined version soon, with the
proper code.
